int32 signature
int32 center_x
int32 center_y
int32 width
int32 height
int32 angle
int32 index
int32 age